### PR TITLE
Weekly Release - 06/14/16 - Enterprise 3.2

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

File: install_config/revhistory_install_config.adoc [hunk 1]
```diff
@@ -6,6 +6,37 @@
...
+|Affected Topic |Description of Change
+//Tue Jun 14 2016
+
+|link:../install_config/aggregate_logging.html#deploying-the-efk-stack[Aggregating Container Logs]
+|Specified the correct units for `*ES_INSTANCE_RAM*` and `*ES_OPS_INSTANCE_RAM*`.
+
+|link:../install_config/storage_examples/privileged_pod_storage.html[Persistent Storage Examples -> Mounting Volumes on Privileged Pods]
+|Added link:../install_config/storage_examples/privileged_pod_storage.html[Mounting Volumes on Privileged Pods] file.
+
+|link:../install_config/install/deploy_router.html[Deploying a Router]
```
Comment (adellape) [inline install_config/revhistory_install_config.adoc]: Missing the `Installing ->`

File: install_config/revhistory_install_config.adoc [hunk 2]
```diff
@@ -6,6 +6,37 @@
...
+|Added an Important box regarding default resource requests for router pods.
+
+|link:../install_config/configuring_authentication.html[Configuring Authentication]
+|Added the `*clientCommonNames*` parameter to the link:../install_config/configuring_authentication.html#RequestHeaderIdentityProvider[Request Header] section.
+
+|link:../install_config/master_node_configuration.html[Master and Node Configuration]
+|Updated the setting guidance in link:../install_config/master_node_configuration.html#master-node-configuration-parallel-image-pulls-with-docker[Parallel Image Pulls with Docker 1.9+].
+
+|link:../install_config/install/docker_registry.html[Deploying a Docker Registry]
```
Comment (adellape) [inline install_config/revhistory_install_config.adoc]: Same for `Installing ->`

File: install_config/revhistory_install_config.adoc [hunk 3]
```diff
@@ -6,6 +6,37 @@
...
+|Updated the example of using an existing persistent volume claim (PVC) to a matching configuration for docker-registry PVC.
```
Comment (adellape) [inline install_config/revhistory_install_config.adoc]: Maybe:

s/docker-registry/the Docker registry PVC/

Comment (adellape) [inline install_config/revhistory_install_config.adoc]: Or markup "docker-registry".

File: using_images/revhistory_using_images.adoc [hunk 4]
```diff
@@ -6,6 +6,26 @@
...
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Tue Jun 14 2016
+
+|link:../using_images/other_images/jenkins.html#versions[Other Images -> Jenkins]
+|Removed specific reference to Jenkins version and added a link to the LTS Changelog.
+
+|link:../using_images/db_images/mongodb.link[Database Images -> MongoDB]
```
Comment (adellape) [inline using_images/revhistory_using_images.adoc]: s/link/html/

**Comment by adellape** [comment]
@ahardin-rh Couple comments, otherwise LGTM.

**Pull request merged**